SteamKit2/SteamKit2/Util/MacHelpers.cs

```diff
@@ -71,7 +71,7 @@ static class LibC
     {
         const string LibraryName = "libc";
 
-        [DllImport(LibraryName, CallingConvention = CallingConvention.Cdecl)]
```

Comment:
All the APIs in this file deal with regular UTF-8 strings (`char *`) and need to be `CharSet.Ansi`. With `CharSet.Unicode`, SK2 just crashes.